x-pack/plugins/uptime/server/lib/requests/get_monitor_duration.ts

```diff
@@ -46,7 +46,7 @@ export const getMonitorDurationChart: UMElasticsearchQueryFn<
           location: {
             terms: {
               field: 'observer.geo.name',
-              missing: 'N/A',
+              missing: UNNAMED_LOCATION,
```

Comment:
What causes an unnamed location? Incorrect configuration? For my edification.

Reply:
when you don't defined a name for a location in hearbeat configuration